### PR TITLE
clarify UI Component flow in README

### PR DESCRIPTION
**Pull request by vhmth**
A little less confusing.

### CODE REVIEW

**Comment by tshaddix** [comment]
@vhmth Sorry I've been away! Going to check this out later today! (it rhymes)

**Comment by vhmth** [comment]
😆 sounds good homie. Waiting on your review ain't slowin' me.

File: README.md [hunk 1]
```diff
@@ -15,9 +15,14 @@ npm install react-chrome-redux
 
 ## Overview
 
-`react-chrome-redux` aims to do one simple thing for Chrome extensions: treat the background script as a single place to store a single source of truth redux Store.
+`react-chrome-redux` aims to do one simple thing for Chrome extensions: treat the background script as a place to store a single source of truth redux Store.
```
Comment (tshaddix) [inline README.md]: I feel like we can weave in a bit of the philosophy discussed in the introduction and #34. What are your thoughts?

Something like:

> `react-chrome-redux` allows you to build your Chrome extension like a Redux-powered webapp. The background page holds the Redux store, while Popovers and Content-Scripts act as UI Components, passing actions and state updates between themselves and the background store. At the end of the day, you have a single source of truth (your Redux store) that describes the entire state of your extension.

File: README.md [hunk 2]
```diff
@@ -15,9 +15,14 @@ npm install react-chrome-redux
...
 
-All UI Components simply pull their react component state from this store on the background and dispatch actions to a store which proxies all actions to the background and receives state updates from the background.
+All UI Components follow the same basic flow:
```
Comment (tshaddix) [inline README.md]: Cool this is much more readable.

File: README.md [hunk 3]
```diff
@@ -15,9 +15,14 @@ npm install react-chrome-redux
...
+
+1. UI Component dispatches action to a Proxy Store.
+2. Proxy Store passes action to background script.
+3. Wrapped Store on the background script updates its state and sends back to UI Component.
```
Comment (tshaddix) [inline README.md]: Having trouble with this wording. At this point in the documentation, "Wrapped Store" doesn't mean anything. Should we simplify this somehow?

Comment (vhmth) [inline README.md]: We could just remove "wrapped". :-)

File: README.md (hunk 1 above)
Comment (vhmth) [inline README.md]: I like it - updating.

**Pull request merged**